wrapper/pre_copy.py

```diff
@@ -137,6 +137,8 @@ def __init__(self, data):
                 self.insecure = False
             else:
                 raise ValueError('Invalid value for "no_verify"')
+        elif data['transport_method'] == 'ssh':
+            self.insecure = True
```

Comment:
Because with SSH URI, there's no `no_verify` parameter. So, we never go into the condition above.
And I'm forcing insecure connection as default in the case of SSH, because `no_verify` is always true with VDDK when called by CloudForms.

Reply:
I understand that the setup most seen in the world is isolated network with self-signed certificates. But defaulting to not even trying to check them, just in case someone as a proper secure setup seems a bit backwards to me.

Reply:
Again, the fact that MIQ/CFME enforces `no_verify` everywhere does not mean it's the only way (especially when in this case it is really bad way).